### DISCUSSION TITLE
Support for non-javascript nx workspaces

### DISCUSSION THREAD

**Discussion by mikehaas763** [nx-dotnet/nx-dotnet#913] [Ideas]
I have a bare minimum non-javascript workspaces, meaning it's setup to use the self-managed nx installation at `.nx/installation`. I went to generate a basic new app, and it failed with "Could not find package.json". I'm assuming this plugin isn't yet aware of this type of nx workspace.

Do we think it would be good to add support for this?

**Comment by AgentEnder** [comment]
Hello - this will be fixed in the @nx/dotnet plugin on release
